### PR TITLE
Added Objetslitt custom catalog

### PR DESCRIPTION
**Pull request by florian-rieder**
Added a new custom catalog for Objetslitt which contains the following features and customizations:

- Added a "Daily item" feature that uses `Rails.cache.fetch` and expires every day at midnight.
- Reorder the simple search result tabs so that the category "textes" will be shown first if available.
- Customized the home page to display a cards menu and the daily item.
- Added a custom page for the "donnees" slug, which displays a simple search form and the daily item.
- Customized the navbar when visiting the "donnees" page to show a different menu specifically for this page.
- Added a mailto icon to the navbar.
- Added logos in the footer.

### CODE REVIEW

File: .gitignore
Comment (lucaguindani) [inline .gitignore:1]: Missing empty new line at the end of the file

File: objetslitt/views/pages/show.html.erb
Comment (lucaguindani) [inline objetslitt/views/pages/show.html.erb:1]: Missing empty new line at the end of the file

File: objetslitt/views/shared/_daily_item.html.erb
Comment (lucaguindani) [inline objetslitt/views/shared/_daily_item.html.erb:1]: Missing empty new line at the end of the file

File: objetslitt/views/shared/navbar/_navbar_catalog.erb
Comment (lucaguindani) [inline objetslitt/views/shared/navbar/_navbar_catalog.erb:1]: Overriding this view to add a menu for a specific page is risky... catalog could break if we make changes to the navbar and cannot backport them to the custom catalog. Maybe you could add the dropdowns **under** the navbar by overriding shared/_page_header.erb (this view contains a lot less logic than the navbar -> fewer risks of breaking) ?

Why absolute urls and not relative ? Remove all the `https://catima.unil.ch` ?

File: objetslitt/views/simple_searches/_list.html.erb
Comment (lucaguindani) [inline objetslitt/views/simple_searches/_list.html.erb:1]: Also risky to override, but this is a feature we're planning to add in the future, so we can keep it like that and we will adapt the catalog when the feature is implemented for everyone.

**Review by lucaguindani — CHANGES_REQUESTED** [review]

File: objetslitt/views/shared/navbar/_navbar_catalog.erb
Comment (florian-rieder) [reply objetslitt/views/shared/navbar/_navbar_catalog.erb:1]: I fully understand the reasons not to override this view. Moving the custom menu for the "donnees" page to the page header is a good idea, beyond avoiding the risky override of navbar_catalog, it also helps with the accessibility and navigability of the catalog by keeping the main menu consistent for every page.

The mailto logo has been moved to the new help_menu view, which avoids overriding navbar_catalog. The absolute urls have been converted to relative.

**Review by lucaguindani — APPROVED** [review]

**Pull request merged**